src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -348,7 +261,7 @@ public String getHubEntityId(MatchingServiceAdapterConfiguration configuration)
     @Singleton
     @Named("AllAcceptableHubConnectorEntityIds")
     public List<String> getAcceptableHubConnectorEntityIds(MatchingServiceAdapterConfiguration configuration) {
```

Comment:
Since there's no more eIDAS this method and its usages can be removed. Or were you planning to do this as part of a separate piece of work?